infra/modules/service/variables.tf

```diff
@@ -51,6 +51,12 @@ variable "subnet_ids" {
   description = "Private subnet id from vpc module"
 }
 
+variable "extra_environment_variables" {
+  type        = map(string)
+  description = "Additional environment variables to pass to the service container"
+  default     = {}
+}
```

Comment:
@rocketnova tagged you on this PR since I ended up needing the extra environment variables functionality so i went ahead and implemented it here